SimpleImageIO/SimpleImageIO.csproj

```diff
@@ -15,7 +15,7 @@
 
     <TargetFramework>net5.0</TargetFramework>
     <!-- Each additional platform requires an entry here, once the native binaries have been added below -->
-    <RuntimeIdentifiers>win10-x64;linux-x64</RuntimeIdentifiers>
+    <RuntimeIdentifiers>win10-x64;linux-x64;osx.10.15-x64</RuntimeIdentifiers>
```

Comment:
Are the macOS 10.15 binaries incompatible with other OS versions? If not, it's probably best to use the more general "osx-64", which corresponds to any version >= macOS 10.12 Sierra

Reply:
Fixed in fa174c4.